pmd-core/src/main/java/net/sourceforge/pmd/renderers/AbstractRenderer.java

```diff
@@ -6,22 +6,31 @@
 
 import java.io.IOException;
 import java.io.Writer;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.List;
 
 import org.apache.commons.io.IOUtils;
 
+import net.sourceforge.pmd.PMDConfiguration;
+import net.sourceforge.pmd.cli.PMDParameters;
 import net.sourceforge.pmd.properties.AbstractPropertySource;
 
 /**
  * Abstract base class for {@link Renderer} implementations.
  */
 public abstract class AbstractRenderer extends AbstractPropertySource implements Renderer {
+    private static final String FILE_SEPARATOR = System.getProperty("file.separator");
```

Comment:
any reason to not use `File.separator` directly?

Reply:
Just reusing the original implementation from DataSource.... but yes, we should simply use File.separator directly :smile: